assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,40 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+let myFunction = () => {};
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
+let anotherFunction = (param => {
+    return param;
+});
 
```

Comment:
the parenthesis here are a little misleading - it kind of looks like you're passing everything into the function as an argument, similar to how you would when calling .map and other array methods. 

you don't need that extra set of parentheses -  you could have something more like "let anotherFunction = (param) => {return param;};" and while it's not my favorite syntax, i just want to make sure you know that when there's a simple single parameter and a simple return like this you can omit the extra and take advantage of the implicit return - "let anotherFunction = param => param" works just fine.